pep/app/styles/components/_ember-table.scss

```diff
@@ -2,10 +2,10 @@
 .ember-table {
     thead {
         th {
-            border-top: 1px solid #fff;
+            border-top: 1px solid $white;
         }
         tr:nth-child(2) > th {
-            border-top: 1px solid #e9e9e9;
+            border-top: 1px solid $table-border;
```

Comment:
as mentioned above, i would just use the existing `$table-border-color` here to match the rest of the table's border color exactly.